zilencer/management/commands/populate_db.py

```diff
@@ -946,8 +946,12 @@ def assign_time_zone_by_delivery_email(delivery_email: str, new_time_zone: str)
                     user=user_profile, onboarding_step="narrow_to_dm_with_welcome_bot_new_user"
                 )
             )
+            onboarding_steps.append(
+                OnboardingStep(user=user_profile, onboarding_step="onboarding_video")
+            )
 
         # Existing users shouldn't narrow to DM with welcome bot on first login.
+        # Also, onboarding video shouldn't be presented to them.
         OnboardingStep.objects.bulk_create(onboarding_steps)
```

Comment:
Probably we should redo these comments to not list all of the items -- just say something like:

"Mark onboarding steps as seen for existing users to avoid unnecessary popups during development."

Reply:
Done.